prow/config.yaml

```diff
@@ -112,6 +112,49 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+    run_after_success:		
+    - name: pull-kubernetes-e2e-gce-canary		
+      agent: kubernetes		
+      always_run: true		
+      context: pull-kubernetes-e2e-gce-canary		
+      rerun_command: "/test pull-kubernetes-e2e-gce-canary"		
+      trigger: "(?m)^/test( all| pull-kubernetes-e2e-gce-canary),?(\\s+|$)"
+      spec:
+        containers:
+        - args:
+          - --bare
```

Comment:
`--bare` and `--repo` don't like each other :imp: get rid of `--bare`

Reply:
done